### DISCUSSION TITLE
Windows arm support for snapdragon chips

### DISCUSSION THREAD

**Discussion by iyashwant** [f1multiviewer/issue-tracker#533] [Q&A] [answered]
Is there any plan for Windows arm support for snapdragon chips based PCs

**Comment by LapsTimeOFF** [comment, marked as answer]
Unfortunately Windows on ARM chips do not support DRM which is required for playing live content from F1 TV. Please fine a bug in Windows feedback asking them to support DRM on ARM devices!
